firewall/app/main/controllers/macController.py

```diff
@@ -36,8 +33,7 @@ def unbanMac():
         response = PyNFT.UnbanMACAddr(address)
         if (response['error']):
             raise Exception(response['error'])
-        MacBan.query.filter_by(address=address).delete()
-        db.session.commit()
+        redis_client.zrem('macBan', address)
```

Comment:
what append if the address is not in the database ?

Reply:
Throw an Exception, with the raise Exception above because the address is not in nftables rules neither